src/static/index.html

```diff
@@ -1,9 +1,10 @@
 <html>
 <head>
-  <script>
-    window.start = new Date().getTime()
-  </script>
-  <script src=" https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js"></script>
-  <script src="./bundle.js"></script>
+    <script>
+        window.start = new Date().getTime()
+    </script>
+    <script src=" https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js"></script>
+    <script src="./bundle.js"></script>
+    <script>main()</script>
```

Comment:
just curious - is this really how to handle this? I thought script tags could be loaded in any order, so you want to put the main() call inside of the caller loading it.

i.e. `<script src="./bundle.js">main()</script>`

If main relies on pyodide, then we should probably use a document.onReady() handler....

Reply:
or use the INIT action, so in main() only reference things available in bundle until after INIT is sent (i.e. no pyodide)

Reply:
I don't have strong feelings about this, it was just a small style experiment. My only thought - which is very, very speculative - is that disassociating the loading of the bundle from calling `main()` means "in the future" you could use a plugin approach here - you want the script for the plugins to load, and then you want to call `main()`, probably providing some argument(s) that confirm to a plugin interface, that allows the plugin functionality to get "linked in". This is all very vague in my mind.

My other reason is that it's just nice to see an explicit invocation of an entrypoint function.

> If main relies on pyodide, then we should probably use a document.onReady() handler....

This is what it does internally - in `main.ts`.

I'm all for whatever arrangement you think is best/most intuitive. Also I sense there may be a naming issue here - maybe `main` is misleading and it should be called `launch()` or `bootstrap()` or ...? Anyway let's talk it over on VC next time.

Reply:
high level though - isn't there a risk main gets called before bundle.js is loaded?

Reply:
Scripts tags are loaded in order